### PR TITLE
npm install waits for nodejs install to finish

### PR DESCRIPTION
**Pull request by nfusionz**

### CODE REVIEW

File: manifests/install.pp [hunk 1]
```diff
@@ -208,11 +208,6 @@
     ensure => running,
     subscribe => [File["/usr/share/jetty9/webapps/blazegraph/WEB-INF/GraphStore.properties"]],
 } ->
-exec { "wait_for_blazegrah":
```
Comment (minorg) [inline manifests/install.pp]: Why was this removed?

Comment (nfusionz) [reply manifests/install.pp]: It didn't work

Comment (minorg) [reply manifests/install.pp]: So the previous ensure is good enough?

The for might need bash -c '' around it, to start a shell, if we still need the curl tries.
